src/app/main/component/places/places.component.spec.ts

```diff
@@ -106,6 +112,18 @@ describe('PlacesComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('ngOnInit should called subscribeToLangChange method one time', () => {
+    const subscribeToLangChangeSpy = spyOn(component as any, 'subscribeToLangChange');
+    component.ngOnInit();
+    expect(subscribeToLangChangeSpy).toHaveBeenCalledTimes(1);
+  });
+
+  it(`bindLang should be called in ngOnInit`, () => {
+    spyOn(component, 'bindLang');
```

Comment:
If you want to cover private method by test write `component as any`

Reply:
Thank you! Changes are edded.